Frontend/app/class/list/list.component.ts

```diff
@@ -31,7 +31,9 @@ export class ClassListComponent implements OnInit {
 
   handleError(err){
     this.isLoading = false;
-    this.toastr.error("The class list failed to load.");
+    if (!err.isAuthenticationError) {
```

Comment:
Talked to you on slack about this. 

__To everyone else :__ Alex was saying that this check wasnt really needed because we should have these things called route guards in place to prevent the user from getting to the page in the first place. There are route guards added as part of this pull request but I added in this additional check in the case of a token expiring or being tampered with. In a lot of cases this check wont be needed since the user shouldnt be able to get to it. But i built it in if anyone wants it later. The `isAuthenticationError` flag will be set for both 401 and 403 response codes.